ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15535236?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15535236?buildTab=artifacts#/) on master @ [3fd77b601d4ccab0f854fef82f9a27b9b3fe303b](https://github.com/cockroachdb/cockroach/commits/3fd77b601d4ccab0f854fef82f9a27b9b3fe303b):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 5210754 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3063082464
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15556610?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15556610?buildTab=artifacts#/) on master @ [7c23978da2866e5fec4ec99b436ca5baa3e4bfd3](https://github.com/cockroachdb/cockroach/commits/7c23978da2866e5fec4ec99b436ca5baa3e4bfd3):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 7357772 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1926158649
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15559455?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15559455?buildTab=artifacts#/) on master @ [e2ddbb469ff4b38f1b38210514eed802073779ed](https://github.com/cockroachdb/cockroach/commits/e2ddbb469ff4b38f1b38210514eed802073779ed):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5761745 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3856072465
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15597092?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15597092?buildTab=artifacts#/) on master @ [9e2e4fbab24c2a75f4c0d0b3a40c3e15e521dd1e](https://github.com/cockroachdb/cockroach/commits/9e2e4fbab24c2a75f4c0d0b3a40c3e15e521dd1e):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5183220 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3928356412
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15617309?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15617309?buildTab=artifacts#/) on master @ [00d41392f1deb1d87e5b9e541f660ec8d04ccbb7](https://github.com/cockroachdb/cockroach/commits/00d41392f1deb1d87e5b9e541f660ec8d04ccbb7):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 7178784 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2910626069
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15635649?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15635649?buildTab=artifacts#/) on master @ [aa0437d20146abd2d9bc2390f34fe3da5ec6bfec](https://github.com/cockroachdb/cockroach/commits/aa0437d20146abd2d9bc2390f34fe3da5ec6bfec):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m50s)

```
Stack: 

```
goroutine 6649591 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1660643343
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15651374?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15651374?buildTab=artifacts#/) on master @ [e1d779792d1126c2f1937e7072f2d9d3776a3308](https://github.com/cockroachdb/cockroach/commits/e1d779792d1126c2f1937e7072f2d9d3776a3308):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 7274821 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1560248042
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15651815?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15651815?buildTab=artifacts#/) on master @ [b1bf99cf6cd6f4ba8e5ed0f87009f0c746b6a8c1](https://github.com/cockroachdb/cockroach/commits/b1bf99cf6cd6f4ba8e5ed0f87009f0c746b6a8c1):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5395776 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality770940385
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15664563?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15664563?buildTab=artifacts#/) on master @ [e6d22a8ffb875d579513fd23fff955ccc495e95f](https://github.com/cockroachdb/cockroach/commits/e6d22a8ffb875d579513fd23fff955ccc495e95f):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5688602 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality350517059
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15700032?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15700032?buildTab=artifacts#/) on master @ [5e92b7997363176d183964703a9ec00ea46188f2](https://github.com/cockroachdb/cockroach/commits/5e92b7997363176d183964703a9ec00ea46188f2):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5776871 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality994583367
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15701055?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15701055?buildTab=artifacts#/) on master @ [5e92b7997363176d183964703a9ec00ea46188f2](https://github.com/cockroachdb/cockroach/commits/5e92b7997363176d183964703a9ec00ea46188f2):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 9031436 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2305746643
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15710322?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15710322?buildTab=artifacts#/) on master @ [d88af9d8d14c906b18a4247ddbdba5f56cba71c9](https://github.com/cockroachdb/cockroach/commits/d88af9d8d14c906b18a4247ddbdba5f56cba71c9):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m51s)

```
Stack: 

```
goroutine 5513201 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2450728679
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15710820?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15710820?buildTab=artifacts#/) on master @ [d88af9d8d14c906b18a4247ddbdba5f56cba71c9](https://github.com/cockroachdb/cockroach/commits/d88af9d8d14c906b18a4247ddbdba5f56cba71c9):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 8220853 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3170100749
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15710841?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15710841?buildTab=artifacts#/) on master @ [d88af9d8d14c906b18a4247ddbdba5f56cba71c9](https://github.com/cockroachdb/cockroach/commits/d88af9d8d14c906b18a4247ddbdba5f56cba71c9):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8984563 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2376973113
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15737044?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15737044?buildTab=artifacts#/) on master @ [dc1c44f2f2d12fe4356139c775aaeff4cff9a4f6](https://github.com/cockroachdb/cockroach/commits/dc1c44f2f2d12fe4356139c775aaeff4cff9a4f6):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5546559 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1346867912
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15740260?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15740260?buildTab=artifacts#/) on master @ [cef0033ce019cc2d37c856bb6e4fe356a0db06cd](https://github.com/cockroachdb/cockroach/commits/cef0033ce019cc2d37c856bb6e4fe356a0db06cd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5673822 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality589842435
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15740326?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15740326?buildTab=artifacts#/) on master @ [cef0033ce019cc2d37c856bb6e4fe356a0db06cd](https://github.com/cockroachdb/cockroach/commits/cef0033ce019cc2d37c856bb6e4fe356a0db06cd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8822268 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality872147895
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15740343?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15740343?buildTab=artifacts#/) on master @ [cef0033ce019cc2d37c856bb6e4fe356a0db06cd](https://github.com/cockroachdb/cockroach/commits/cef0033ce019cc2d37c856bb6e4fe356a0db06cd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 9268503 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality411748073
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15761114?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15761114?buildTab=artifacts#/) on master @ [ad609dd3657ad4975ce47a71bd65f3eeb9cd8f92](https://github.com/cockroachdb/cockroach/commits/ad609dd3657ad4975ce47a71bd65f3eeb9cd8f92):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5773194 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1057920893
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15762536?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15762536?buildTab=artifacts#/) on master @ [7a22234cc9345bf3ffeae2f638c7f82369b09aea](https://github.com/cockroachdb/cockroach/commits/7a22234cc9345bf3ffeae2f638c7f82369b09aea):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 4988122 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality100901059
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15763388?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15763388?buildTab=artifacts#/) on master @ [7a22234cc9345bf3ffeae2f638c7f82369b09aea](https://github.com/cockroachdb/cockroach/commits/7a22234cc9345bf3ffeae2f638c7f82369b09aea):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8734679 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality4214304170
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15772265?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15772265?buildTab=artifacts#/) on master @ [4a708c864424fd0163738869ab2602a958ad2c09](https://github.com/cockroachdb/cockroach/commits/4a708c864424fd0163738869ab2602a958ad2c09):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 4529946 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality4171718026
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15773006?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15773006?buildTab=artifacts#/) on master @ [8a97a5edd98336e2dd04ef12f08628fba84b17dd](https://github.com/cockroachdb/cockroach/commits/8a97a5edd98336e2dd04ef12f08628fba84b17dd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 8881517 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3544786777
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15773397?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15773397?buildTab=artifacts#/) on master @ [8a97a5edd98336e2dd04ef12f08628fba84b17dd](https://github.com/cockroachdb/cockroach/commits/8a97a5edd98336e2dd04ef12f08628fba84b17dd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 8694951 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2158852957
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15773573?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15773573?buildTab=artifacts#/) on master @ [8a97a5edd98336e2dd04ef12f08628fba84b17dd](https://github.com/cockroachdb/cockroach/commits/8a97a5edd98336e2dd04ef12f08628fba84b17dd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 7197041 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1384530501
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15773760?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15773760?buildTab=artifacts#/) on master @ [8a97a5edd98336e2dd04ef12f08628fba84b17dd](https://github.com/cockroachdb/cockroach/commits/8a97a5edd98336e2dd04ef12f08628fba84b17dd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 9537127 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality42547498
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15774105?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15774105?buildTab=artifacts#/) on master @ [8a97a5edd98336e2dd04ef12f08628fba84b17dd](https://github.com/cockroachdb/cockroach/commits/8a97a5edd98336e2dd04ef12f08628fba84b17dd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 9113751 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality4092735250
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15774280?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15774280?buildTab=artifacts#/) on master @ [8a97a5edd98336e2dd04ef12f08628fba84b17dd](https://github.com/cockroachdb/cockroach/commits/8a97a5edd98336e2dd04ef12f08628fba84b17dd):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8784295 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1416339480
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15825957?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15825957?buildTab=artifacts#/) on master @ [3b03505a3b32cafb300583203caa1f82297904e0](https://github.com/cockroachdb/cockroach/commits/3b03505a3b32cafb300583203caa1f82297904e0):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m51s)

```
Stack: 

```
goroutine 7848937 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3091668817
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15828862?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15828862?buildTab=artifacts#/) on master @ [361f58bdb476e76fd3c6c9a2f54a48b7c74e9797](https://github.com/cockroachdb/cockroach/commits/361f58bdb476e76fd3c6c9a2f54a48b7c74e9797):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 6388260 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2230412106
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15836235?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15836235?buildTab=artifacts#/) on master @ [e7db4a236a90666423976870b413e7551711b9d3](https://github.com/cockroachdb/cockroach/commits/e7db4a236a90666423976870b413e7551711b9d3):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8146982 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality179656683
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15860363?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15860363?buildTab=artifacts#/) on master @ [b5ece5bd70bc5d82662d055ca4b2dd5d34a2316d](https://github.com/cockroachdb/cockroach/commits/b5ece5bd70bc5d82662d055ca4b2dd5d34a2316d):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5802818 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1328161508
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15863886?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15863886?buildTab=artifacts#/) on master @ [23fdf84e0848e8a91d9f2cd5053568a7ad1e7425](https://github.com/cockroachdb/cockroach/commits/23fdf84e0848e8a91d9f2cd5053568a7ad1e7425):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 7760395 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality913655383
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15864933?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15864933?buildTab=artifacts#/) on master @ [ff679af10fb499cc9943ac33ee11159e5434d3ce](https://github.com/cockroachdb/cockroach/commits/ff679af10fb499cc9943ac33ee11159e5434d3ce):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 4588292 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2351334083
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15865586?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15865586?buildTab=artifacts#/) on master @ [aaec845947bbf7242e7ce512e2bfe338106a0876](https://github.com/cockroachdb/cockroach/commits/aaec845947bbf7242e7ce512e2bfe338106a0876):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m50s)

```
Stack: 

```
goroutine 6117211 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1390820127
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15866623?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15866623?buildTab=artifacts#/) on master @ [047a7ed79756eef53b8b9ab4c9dd9c5a463496c9](https://github.com/cockroachdb/cockroach/commits/047a7ed79756eef53b8b9ab4c9dd9c5a463496c9):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 6282525 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3696172336
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15888577?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15888577?buildTab=artifacts#/) on master @ [9c008f5b8c21fdffafa22bfd6c026f5f1c642e06](https://github.com/cockroachdb/cockroach/commits/9c008f5b8c21fdffafa22bfd6c026f5f1c642e06):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 8445487 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3668796760
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15911433?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15911433?buildTab=artifacts#/) on master @ [65a16176c8926a35399a1a7dd69d1e23a70a8542](https://github.com/cockroachdb/cockroach/commits/65a16176c8926a35399a1a7dd69d1e23a70a8542):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 6572468 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3795852008
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15921794?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15921794?buildTab=artifacts#/) on master @ [5afdc46fe5b8e91a087c7170f0acd43503f13497](https://github.com/cockroachdb/cockroach/commits/5afdc46fe5b8e91a087c7170f0acd43503f13497):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 6614782 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3472188522
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15982228?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15982228?buildTab=artifacts#/) on master @ [a1091c28c6d9a2e8ab88c0f82624e4c9e11cd483](https://github.com/cockroachdb/cockroach/commits/a1091c28c6d9a2e8ab88c0f82624e4c9e11cd483):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m45s)

```
Stack: 

```
goroutine 8303834 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality72126586
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15987474?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15987474?buildTab=artifacts#/) on master @ [286cb2ca47bb817b038de1868f55fa1127509802](https://github.com/cockroachdb/cockroach/commits/286cb2ca47bb817b038de1868f55fa1127509802):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 3991004 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2683916972
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15989110?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15989110?buildTab=artifacts#/) on master @ [c80d79c4b8d76d6628972381ae0d9f564dedc86a](https://github.com/cockroachdb/cockroach/commits/c80d79c4b8d76d6628972381ae0d9f564dedc86a):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5459300 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1742688891
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15989425?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15989425?buildTab=artifacts#/) on master @ [c80d79c4b8d76d6628972381ae0d9f564dedc86a](https://github.com/cockroachdb/cockroach/commits/c80d79c4b8d76d6628972381ae0d9f564dedc86a):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8916184 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality62175749
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15989990?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15989990?buildTab=artifacts#/) on master @ [6b22014dec8c5e8e6006d710ed05bab5b01b4ae2](https://github.com/cockroachdb/cockroach/commits/6b22014dec8c5e8e6006d710ed05bab5b01b4ae2):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8699038 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1387817802
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15997085?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15997085?buildTab=artifacts#/) on master @ [10981d254d1b826f93b5b3a6bfdf30d27ff2a755](https://github.com/cockroachdb/cockroach/commits/10981d254d1b826f93b5b3a6bfdf30d27ff2a755):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5829686 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality767526275
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15998299?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15998299?buildTab=artifacts#/) on master @ [46a5cb142d7f9c19c190d63dc7c8f91195e4ef3c](https://github.com/cockroachdb/cockroach/commits/46a5cb142d7f9c19c190d63dc7c8f91195e4ef3c):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 7591874 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2800345288
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16015193?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16015193?buildTab=artifacts#/) on master @ [551dcfbaf516139ab8c6f5d30269a00b0bdce032](https://github.com/cockroachdb/cockroach/commits/551dcfbaf516139ab8c6f5d30269a00b0bdce032):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8269123 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality469915232
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16016404?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16016404?buildTab=artifacts#/) on master @ [551dcfbaf516139ab8c6f5d30269a00b0bdce032](https://github.com/cockroachdb/cockroach/commits/551dcfbaf516139ab8c6f5d30269a00b0bdce032):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 9125079 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3237951082
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16016795?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16016795?buildTab=artifacts#/) on master @ [551dcfbaf516139ab8c6f5d30269a00b0bdce032](https://github.com/cockroachdb/cockroach/commits/551dcfbaf516139ab8c6f5d30269a00b0bdce032):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 8923201 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality211244607
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16036882?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16036882?buildTab=artifacts#/) on master @ [c7e7b1997b7d39342bc22fc7f2dfd3df301c7d4f](https://github.com/cockroachdb/cockroach/commits/c7e7b1997b7d39342bc22fc7f2dfd3df301c7d4f):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 6120009 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2852146307
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16041955?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16041955?buildTab=artifacts#/) on master @ [272308f2498890129ab370f8a67d57ac26d9d4f3](https://github.com/cockroachdb/cockroach/commits/272308f2498890129ab370f8a67d57ac26d9d4f3):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m51s)

```
Stack: 

```
goroutine 7061386 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2025274711
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16079794?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16079794?buildTab=artifacts#/) on master @ [c632eb6fcff4312cbdd8d3ace399a20091a840d5](https://github.com/cockroachdb/cockroach/commits/c632eb6fcff4312cbdd8d3ace399a20091a840d5):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5410352 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality140723206
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16085211?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16085211?buildTab=artifacts#/) on master @ [6cd8823ad8bf25a63e463cc5c0a9a60df9b29cd8](https://github.com/cockroachdb/cockroach/commits/6cd8823ad8bf25a63e463cc5c0a9a60df9b29cd8):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m51s)

```
Stack: 

```
goroutine 6464626 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2793145890
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16100194?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16100194?buildTab=artifacts#/) on master @ [475450da7c921e79716cea0a4c45fc75cd34735c](https://github.com/cockroachdb/cockroach/commits/475450da7c921e79716cea0a4c45fc75cd34735c):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 4755927 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality881564446
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot T-cdc branch-release-24.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16107888?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16107888?buildTab=artifacts#/) on master @ [dca553c77b8d9a2a8a6b9bfc3ead1b1bd2f4babb](https://github.com/cockroachdb/cockroach/commits/dca553c77b8d9a2a8a6b9bfc3ead1b1bd2f4babb):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 5101801 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3802131864
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16118117?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16118117?buildTab=artifacts#/) on master @ [9a78f2708ac779a34f77d73f52a269b487d30bdc](https://github.com/cockroachdb/cockroach/commits/9a78f2708ac779a34f77d73f52a269b487d30bdc):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 3848627 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1143771637
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16134567?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16134567?buildTab=artifacts#/) on master @ [ffcbb140032238f9111c8457dd3128380d724c79](https://github.com/cockroachdb/cockroach/commits/ffcbb140032238f9111c8457dd3128380d724c79):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 5464088 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality454866362
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16148848?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16148848?buildTab=artifacts#/) on master @ [5e04890407951f0d05c8b015e766619257ef0712](https://github.com/cockroachdb/cockroach/commits/5e04890407951f0d05c8b015e766619257ef0712):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 6952915 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2165623043
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16149522?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16149522?buildTab=artifacts#/) on master @ [5e04890407951f0d05c8b015e766619257ef0712](https://github.com/cockroachdb/cockroach/commits/5e04890407951f0d05c8b015e766619257ef0712):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 8743962 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3796720258
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16149771?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16149771?buildTab=artifacts#/) on master @ [5e04890407951f0d05c8b015e766619257ef0712](https://github.com/cockroachdb/cockroach/commits/5e04890407951f0d05c8b015e766619257ef0712):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8773334 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality526764416
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16150618?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16150618?buildTab=artifacts#/) on master @ [ac0b25c448705b6e2e549ac9e4bf8f24f809d510](https://github.com/cockroachdb/cockroach/commits/ac0b25c448705b6e2e549ac9e4bf8f24f809d510):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5077243 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3460789749
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16162019?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16162019?buildTab=artifacts#/) on master @ [078bc26df0d40231ccf0b1b8332fa1066742000f](https://github.com/cockroachdb/cockroach/commits/078bc26df0d40231ccf0b1b8332fa1066742000f):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 7071178 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3535873336
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16169013?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16169013?buildTab=artifacts#/) on master @ [05052df0b4d89dd1147714431163cd8a9b63c589](https://github.com/cockroachdb/cockroach/commits/05052df0b4d89dd1147714431163cd8a9b63c589):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m49s)

```
Stack: 

```
goroutine 5673220 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1256297118
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16169614?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16169614?buildTab=artifacts#/) on master @ [05052df0b4d89dd1147714431163cd8a9b63c589](https://github.com/cockroachdb/cockroach/commits/05052df0b4d89dd1147714431163cd8a9b63c589):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 9297767 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3837579665
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16185446?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16185446?buildTab=artifacts#/) on master @ [e8357e71cb0a68af7a085f8bb8a0000cdeac2aa4](https://github.com/cockroachdb/cockroach/commits/e8357e71cb0a68af7a085f8bb8a0000cdeac2aa4):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5547883 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality945626722
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16186256?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16186256?buildTab=artifacts#/) on master @ [e8357e71cb0a68af7a085f8bb8a0000cdeac2aa4](https://github.com/cockroachdb/cockroach/commits/e8357e71cb0a68af7a085f8bb8a0000cdeac2aa4):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 8761130 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1729968122
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16186527?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16186527?buildTab=artifacts#/) on master @ [e8357e71cb0a68af7a085f8bb8a0000cdeac2aa4](https://github.com/cockroachdb/cockroach/commits/e8357e71cb0a68af7a085f8bb8a0000cdeac2aa4):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m51s)

```
Stack: 

```
goroutine 4712615 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality655761694
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16200117?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16200117?buildTab=artifacts#/) on master @ [1fa0970c0d48246ee19080569960987b407f69ca](https://github.com/cockroachdb/cockroach/commits/1fa0970c0d48246ee19080569960987b407f69ca):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m51s)

```
Stack: 

```
goroutine 8941722 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2799017313
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16208770?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16208770?buildTab=artifacts#/) on master @ [da85c982fa391ef34fd14c29006b1f629217ff42](https://github.com/cockroachdb/cockroach/commits/da85c982fa391ef34fd14c29006b1f629217ff42):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 8037898 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality966247545
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16227729?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16227729?buildTab=artifacts#/) on master @ [9938507f0dcdf9796d6085d7d5303f4659b4aa09](https://github.com/cockroachdb/cockroach/commits/9938507f0dcdf9796d6085d7d5303f4659b4aa09):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m49s)

```
Stack: 

```
goroutine 6367744 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality161997168
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16249677?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16249677?buildTab=artifacts#/) on master @ [26ce6ee1bf3c626ef72f673770cb42e814b7bd9d](https://github.com/cockroachdb/cockroach/commits/26ce6ee1bf3c626ef72f673770cb42e814b7bd9d):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m49s)

```
Stack: 

```
goroutine 5599878 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1898102072
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16252337?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16252337?buildTab=artifacts#/) on master @ [36f1b94e2000d286716bac8fcdb355f5ed35bacb](https://github.com/cockroachdb/cockroach/commits/36f1b94e2000d286716bac8fcdb355f5ed35bacb):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 8339740 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2907397406
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16254853?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16254853?buildTab=artifacts#/) on master @ [88594c40fa36a654c63ae78eb2f1aea57d6af773](https://github.com/cockroachdb/cockroach/commits/88594c40fa36a654c63ae78eb2f1aea57d6af773):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 6472524 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1906358566
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16257323?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16257323?buildTab=artifacts#/) on master @ [5d9405ff2e16622c055f94208992a5347c4c273e](https://github.com/cockroachdb/cockroach/commits/5d9405ff2e16622c055f94208992a5347c4c273e):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 8164837 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3603138354
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16263590?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16263590?buildTab=artifacts#/) on master @ [0541689cffc63878b387aff62e7ed1966d2ff79a](https://github.com/cockroachdb/cockroach/commits/0541689cffc63878b387aff62e7ed1966d2ff79a):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m50s)

```
Stack: 

```
goroutine 4941816 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3096948367
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16275334?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16275334?buildTab=artifacts#/) on master @ [ba114ad0a655e6ce97c58303fea71857689ce240](https://github.com/cockroachdb/cockroach/commits/ba114ad0a655e6ce97c58303fea71857689ce240):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m51s)

```
Stack: 

```
goroutine 6849469 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3109976305
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16283204?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16283204?buildTab=artifacts#/) on master @ [173daef28330f3c8fd1a3c361a239734078fabc3](https://github.com/cockroachdb/cockroach/commits/173daef28330f3c8fd1a3c361a239734078fabc3):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 5735349 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1215284843
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16290842?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16290842?buildTab=artifacts#/) on master @ [bca0e6cc184be2838468d52733ac3adfbe451a36](https://github.com/cockroachdb/cockroach/commits/bca0e6cc184be2838468d52733ac3adfbe451a36):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 6502520 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1147357898
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16291533?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16291533?buildTab=artifacts#/) on master @ [f9b40638d2f50aa3dd2a32d3a94b460940276c2b](https://github.com/cockroachdb/cockroach/commits/f9b40638d2f50aa3dd2a32d3a94b460940276c2b):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 6830412 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality413274967
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16294354?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16294354?buildTab=artifacts#/) on master @ [62a0e6297cb41142fe99dcd9846ff78e36e3ec69](https://github.com/cockroachdb/cockroach/commits/62a0e6297cb41142fe99dcd9846ff78e36e3ec69):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5296736 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1823143657
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16297703?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16297703?buildTab=artifacts#/) on master @ [78fa126dfc749e620c28531c1659e35bf98034ec](https://github.com/cockroachdb/cockroach/commits/78fa126dfc749e620c28531c1659e35bf98034ec):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 4928174 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2022138673
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16298020?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16298020?buildTab=artifacts#/) on master @ [78fa126dfc749e620c28531c1659e35bf98034ec](https://github.com/cockroachdb/cockroach/commits/78fa126dfc749e620c28531c1659e35bf98034ec):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8830235 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality4285288346
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16298084?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16298084?buildTab=artifacts#/) on master @ [78fa126dfc749e620c28531c1659e35bf98034ec](https://github.com/cockroachdb/cockroach/commits/78fa126dfc749e620c28531c1659e35bf98034ec):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 10274418 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1347821096
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16307668?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16307668?buildTab=artifacts#/) on master @ [c35f564680f650645a8660fe87efd30a4ad72e29](https://github.com/cockroachdb/cockroach/commits/c35f564680f650645a8660fe87efd30a4ad72e29):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 6027751 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2700223083
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16308330?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16308330?buildTab=artifacts#/) on master @ [823e37e3cc4fdcc4a8f8bb5a04fb337d74bd798c](https://github.com/cockroachdb/cockroach/commits/823e37e3cc4fdcc4a8f8bb5a04fb337d74bd798c):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5571890 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1283201385
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16312844?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16312844?buildTab=artifacts#/) on master @ [ed0cc29ced53233f0b686ed5f474ae4035fec3c4](https://github.com/cockroachdb/cockroach/commits/ed0cc29ced53233f0b686ed5f474ae4035fec3c4):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 6604276 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality4053784039
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16340042?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16340042?buildTab=artifacts#/) on master @ [1f3ae113080650426a88f97723d4b3d7addd64ca](https://github.com/cockroachdb/cockroach/commits/1f3ae113080650426a88f97723d4b3d7addd64ca):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 8842655 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality870480248
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16342298?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16342298?buildTab=artifacts#/) on master @ [3173e63fac9f4d3f0632ac6113bf6273451d9f00](https://github.com/cockroachdb/cockroach/commits/3173e63fac9f4d3f0632ac6113bf6273451d9f00):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5964740 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality4157043758
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16349586?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16349586?buildTab=artifacts#/) on master @ [7ea67ccd150cac6fca8f464ba417876ecce007ff](https://github.com/cockroachdb/cockroach/commits/7ea67ccd150cac6fca8f464ba417876ecce007ff):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 4983337 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3532486171
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16350990?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16350990?buildTab=artifacts#/) on master @ [dc4552eac4e5109846b2435738d78422e64b70c8](https://github.com/cockroachdb/cockroach/commits/dc4552eac4e5109846b2435738d78422e64b70c8):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 5109240 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality4189485184
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16351384?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16351384?buildTab=artifacts#/) on master @ [e77f7145617fe9a14bb3e50b032f6804d4459183](https://github.com/cockroachdb/cockroach/commits/e77f7145617fe9a14bb3e50b032f6804d4459183):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 5751060 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2781960200
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16354484?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16354484?buildTab=artifacts#/) on master @ [2584c5a57071b491535e7d426f5bb26feb781690](https://github.com/cockroachdb/cockroach/commits/2584c5a57071b491535e7d426f5bb26feb781690):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5729866 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3476110946
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:407: creating and splitting table into single-key ranges
    changefeed_dist_test.go:421: relocating range for 0 to store 1
    changefeed_dist_test.go:421: relocating range for 1 to store 1
    changefeed_dist_test.go:421: relocating range for 2 to store 2
    changefeed_dist_test.go:421: relocating range for 3 to store 2
    changefeed_dist_test.go:421: relocating range for 4 to store 3
    changefeed_dist_test.go:421: relocating range for 5 to store 3
    changefeed_dist_test.go:421: relocating range for 6 to store 3
    changefeed_dist_test.go:421: relocating range for 7 to store 3
    changefeed_dist_test.go:421: relocating range for 8 to store 4
    changefeed_dist_test.go:421: relocating range for 9 to store 4
    changefeed_dist_test.go:421: relocating range for 10 to store 4
    changefeed_dist_test.go:421: relocating range for 11 to store 4
    changefeed_dist_test.go:421: relocating range for 12 to store 4
    changefeed_dist_test.go:421: relocating range for 13 to store 4
    changefeed_dist_test.go:421: relocating range for 14 to store 4
    changefeed_dist_test.go:421: relocating range for 15 to store 4
    changefeed_dist_test.go:421: relocating range for 16 to store 5
    changefeed_dist_test.go:421: relocating range for 17 to store 5
    changefeed_dist_test.go:421: relocating range for 18 to store 5
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16355054?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16355054?buildTab=artifacts#/) on master @ [1e71de11f35751dd6e7cd28ef2284ee2d14329b8](https://github.com/cockroachdb/cockroach/commits/1e71de11f35751dd6e7cd28ef2284ee2d14329b8):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m49s)

```
Stack: 

```
goroutine 6500592 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1838691784
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:407: creating and splitting table into single-key ranges
    changefeed_dist_test.go:421: relocating range for 0 to store 1
    changefeed_dist_test.go:421: relocating range for 1 to store 1
    changefeed_dist_test.go:421: relocating range for 2 to store 2
    changefeed_dist_test.go:421: relocating range for 3 to store 2
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16366144?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16366144?buildTab=artifacts#/) on master @ [98a6c1bfa342f90b8929911afbc2f50df3173f69](https://github.com/cockroachdb/cockroach/commits/98a6c1bfa342f90b8929911afbc2f50df3173f69):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 5817439 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3143662867
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:407: creating and splitting table into single-key ranges
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16369236?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16369236?buildTab=artifacts#/) on master @ [ba64263d191e298b061b900c174d0b035b6b1cdb](https://github.com/cockroachdb/cockroach/commits/ba64263d191e298b061b900c174d0b035b6b1cdb):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 6849690 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1424923822
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:407: creating and splitting table into single-key ranges
    changefeed_dist_test.go:421: relocating range for 0 to store 1
    changefeed_dist_test.go:421: relocating range for 1 to store 1
    changefeed_dist_test.go:421: relocating range for 2 to store 2
    changefeed_dist_test.go:421: relocating range for 3 to store 2
    changefeed_dist_test.go:421: relocating range for 4 to store 3
    changefeed_dist_test.go:421: relocating range for 5 to store 3
    changefeed_dist_test.go:421: relocating range for 6 to store 3
    changefeed_dist_test.go:421: relocating range for 7 to store 3
    changefeed_dist_test.go:421: relocating range for 8 to store 4
    changefeed_dist_test.go:421: relocating range for 9 to store 4
    changefeed_dist_test.go:421: relocating range for 10 to store 4
    changefeed_dist_test.go:421: relocating range for 11 to store 4
    changefeed_dist_test.go:421: relocating range for 12 to store 4
    changefeed_dist_test.go:421: relocating range for 13 to store 4
    changefeed_dist_test.go:421: relocating range for 14 to store 4
    changefeed_dist_test.go:421: relocating range for 15 to store 4
    changefeed_dist_test.go:421: relocating range for 16 to store 5
    changefeed_dist_test.go:421: relocating range for 17 to store 5
    changefeed_dist_test.go:421: relocating range for 18 to store 5
    changefeed_dist_test.go:421: relocating range for 19 to store 5
    changefeed_dist_test.go:421: relocating range for 20 to store 5
    changefeed_dist_test.go:421: relocating range for 21 to store 5
    changefeed_dist_test.go:421: relocating range for 22 to store 5
    changefeed_dist_test.go:421: relocating range for 23 to store 5
    changefeed_dist_test.go:421: relocating range for 24 to store 5
    changefeed_dist_test.go:421: relocating range for 25 to store 5
    changefeed_dist_test.go:421: relocating range for 26 to store 5
    changefeed_dist_test.go:421: relocating range for 27 to store 5
    changefeed_dist_test.go:421: relocating range for 28 to store 5
    changefeed_dist_test.go:421: relocating range for 29 to store 5
    changefeed_dist_test.go:421: relocating range for 30 to store 5
    changefeed_dist_test.go:421: relocating range for 31 to store 5
    changefeed_dist_test.go:421: relocating range for 32 to store 6
    changefeed_dist_test.go:421: relocating range for 33 to store 6
    changefeed_dist_test.go:421: relocating range for 34 to store 6
    changefeed_dist_test.go:421: relocating range for 35 to store 6
    changefeed_dist_test.go:421: relocating range for 36 to store 6
    changefeed_dist_test.go:421: relocating range for 37 to store 6
    changefeed_dist_test.go:421: relocating range for 38 to store 6
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16370105?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16370105?buildTab=artifacts#/) on master @ [51b3334c7408fb70ba6be7ff717ff650ff1b7f5e](https://github.com/cockroachdb/cockroach/commits/51b3334c7408fb70ba6be7ff717ff650ff1b7f5e):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5041199 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3891310379
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:407: creating and splitting table into single-key ranges
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16376960?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16376960?buildTab=artifacts#/) on master @ [1e912ee64563b84eb0573ab84304c2a248036813](https://github.com/cockroachdb/cockroach/commits/1e912ee64563b84eb0573ab84304c2a248036813):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 4394071 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality893765817
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:407: creating and splitting table into single-key ranges
    changefeed_dist_test.go:421: relocating range for 0 to store 1
    changefeed_dist_test.go:421: relocating range for 1 to store 1
    changefeed_dist_test.go:421: relocating range for 2 to store 2
    changefeed_dist_test.go:421: relocating range for 3 to store 2
    changefeed_dist_test.go:421: relocating range for 4 to store 3
    changefeed_dist_test.go:421: relocating range for 5 to store 3
    changefeed_dist_test.go:421: relocating range for 6 to store 3
    changefeed_dist_test.go:421: relocating range for 7 to store 3
    changefeed_dist_test.go:421: relocating range for 8 to store 4
    changefeed_dist_test.go:421: relocating range for 9 to store 4
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16383123?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16383123?buildTab=artifacts#/) on master @ [bef28ed9cd6c6f712613496ea97bd6a2961e1622](https://github.com/cockroachdb/cockroach/commits/bef28ed9cd6c6f712613496ea97bd6a2961e1622):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 8450047 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality196542765
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:407: creating and splitting table into single-key ranges
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16385708?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16385708?buildTab=artifacts#/) on master @ [769d76a90001dc97dc6d1a9c73ecde1e5242ce9e](https://github.com/cockroachdb/cockroach/commits/769d76a90001dc97dc6d1a9c73ecde1e5242ce9e):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8690904 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality215386665
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:396: starting the test cluster took 15.11915323s
    changefeed_dist_test.go:411: toggling replicate queues off took 18.317µs
    changefeed_dist_test.go:413: creating and splitting table into single-key ranges
    changefeed_dist_test.go:419: creating and inserting into table took 73.116038ms
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16387217?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16387217?buildTab=artifacts#/) on master @ [8a21ac3940ab440030467ebe2feb7abde1030145](https://github.com/cockroachdb/cockroach/commits/8a21ac3940ab440030467ebe2feb7abde1030145):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5405876 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3465678434
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:396: starting the test cluster took 16.723462946s
    changefeed_dist_test.go:411: toggling replicate queues off took 22.844µs
    changefeed_dist_test.go:413: creating and splitting table into single-key ranges
    changefeed_dist_test.go:419: creating and inserting into table took 258.513675ms
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16391889?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16391889?buildTab=artifacts#/) on master @ [b235685babde5ffe4ee0508039cb83e32fedaf09](https://github.com/cockroachdb/cockroach/commits/b235685babde5ffe4ee0508039cb83e32fedaf09):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5706319 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 3 to store 2 took 84.938691ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 4 to store 3
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 4 to store 3 took 67.042684ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 5 to store 3
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 5 to store 3 took 35.258159ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 6 to store 3
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 6 to store 3 took 127.072629ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 7 to store 3
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 7 to store 3 took 95.820203ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 8 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 8 to store 4 took 81.037365ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 9 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 9 to store 4 took 44.89942ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 10 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 10 to store 4 took 40.453743ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 11 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 11 to store 4 took 121.645537ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 12 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:12:57: relocating range for 12 to store 4 took 84.675744ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:57: relocating range for 13 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 13 to store 4 took 85.011418ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 14 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 14 to store 4 took 158.891138ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 15 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 15 to store 4 took 60.784692ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 16 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 16 to store 5 took 37.14885ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 17 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 17 to store 5 took 56.983622ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 18 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 18 to store 5 took 33.782259ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 19 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 19 to store 5 took 50.301908ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 20 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 20 to store 5 took 89.803208ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 21 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 21 to store 5 took 96.202879ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 22 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 22 to store 5 took 85.194516ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 23 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 23 to store 5 took 28.976561ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 24 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 24 to store 5 took 87.094462ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 25 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:58: relocating range for 25 to store 5 took 83.697568ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:58: relocating range for 26 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:59: relocating range for 26 to store 5 took 63.494818ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:59: relocating range for 27 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:12:59: relocating range for 27 to store 5 took 77.326353ms
    changefeed_dist_test.go:432: 2024-08-09 12:12:59: relocating range for 28 to store 5
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16392180?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16392180?buildTab=artifacts#/) on master @ [eb1193e3a8318f36291117ba5df3005e66859221](https://github.com/cockroachdb/cockroach/commits/eb1193e3a8318f36291117ba5df3005e66859221):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 6569438 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:437: 2024-08-09 12:35:59: relocating range for 2 to store 2 took 92.563145ms
    changefeed_dist_test.go:432: 2024-08-09 12:35:59: relocating range for 3 to store 2
    changefeed_dist_test.go:437: 2024-08-09 12:35:59: relocating range for 3 to store 2 took 37.333328ms
    changefeed_dist_test.go:432: 2024-08-09 12:35:59: relocating range for 4 to store 3
    changefeed_dist_test.go:437: 2024-08-09 12:35:59: relocating range for 4 to store 3 took 143.107593ms
    changefeed_dist_test.go:432: 2024-08-09 12:35:59: relocating range for 5 to store 3
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 5 to store 3 took 80.829456ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 6 to store 3
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 6 to store 3 took 74.368924ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 7 to store 3
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 7 to store 3 took 64.993078ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 8 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 8 to store 4 took 48.484138ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 9 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 9 to store 4 took 73.355778ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 10 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 10 to store 4 took 91.483024ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 11 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 11 to store 4 took 135.049421ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 12 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 12 to store 4 took 41.586274ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 13 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 13 to store 4 took 113.550654ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 14 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 14 to store 4 took 28.084723ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 15 to store 4
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 15 to store 4 took 34.809948ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 16 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 16 to store 5 took 135.455879ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 17 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:00: relocating range for 17 to store 5 took 82.670556ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:00: relocating range for 18 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 18 to store 5 took 109.791474ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 19 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 19 to store 5 took 105.288048ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 20 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 20 to store 5 took 44.148759ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 21 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 21 to store 5 took 45.3901ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 22 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 22 to store 5 took 42.631833ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 23 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 23 to store 5 took 29.836032ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 24 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 24 to store 5 took 40.190954ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 25 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 25 to store 5 took 40.492789ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 26 to store 5
    changefeed_dist_test.go:437: 2024-08-09 12:36:01: relocating range for 26 to store 5 took 202.129899ms
    changefeed_dist_test.go:432: 2024-08-09 12:36:01: relocating range for 27 to store 5
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16393099?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16393099?buildTab=artifacts#/) on master @ [eb1193e3a8318f36291117ba5df3005e66859221](https://github.com/cockroachdb/cockroach/commits/eb1193e3a8318f36291117ba5df3005e66859221):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 8795705 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 32 to store 6 took 26.14162ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 33 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 33 to store 6 took 25.638583ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 34 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 34 to store 6 took 25.368899ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 35 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 35 to store 6 took 25.468009ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 36 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 36 to store 6 took 26.356626ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 37 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 37 to store 6 took 25.494743ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 38 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 38 to store 6 took 25.612559ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 39 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 39 to store 6 took 25.756945ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 40 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 40 to store 6 took 30.187612ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 41 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 41 to store 6 took 28.904458ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 42 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 42 to store 6 took 26.130034ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 43 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 43 to store 6 took 27.328593ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 44 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 44 to store 6 took 26.321779ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 45 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 45 to store 6 took 26.318683ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 46 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 46 to store 6 took 26.204308ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 47 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 47 to store 6 took 26.549976ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 48 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 48 to store 6 took 26.253446ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 49 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 49 to store 6 took 26.320113ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 50 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 50 to store 6 took 26.168677ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 51 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 51 to store 6 took 26.128519ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 52 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 52 to store 6 took 30.100923ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 53 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 53 to store 6 took 26.760353ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 54 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 54 to store 6 took 26.606898ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 55 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 55 to store 6 took 26.813837ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 56 to store 6
    changefeed_dist_test.go:437: 2024-08-09 13:09:47: relocating range for 56 to store 6 took 26.17064ms
    changefeed_dist_test.go:432: 2024-08-09 13:09:47: relocating range for 57 to store 6
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16398043?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16398043?buildTab=artifacts#/) on master @ [22c9923eb9854b9e31068234599edce7c856fbde](https://github.com/cockroachdb/cockroach/commits/22c9923eb9854b9e31068234599edce7c856fbde):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5188925 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:437: 2024-08-09 18:10:15: relocating range for 6 to store 3 took 297.897269ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:15: relocating range for 7 to store 3
    changefeed_dist_test.go:437: 2024-08-09 18:10:26: relocating range for 7 to store 3 took 10.907825254s
    changefeed_dist_test.go:432: 2024-08-09 18:10:26: relocating range for 8 to store 4
    changefeed_dist_test.go:437: 2024-08-09 18:10:27: relocating range for 8 to store 4 took 472.835019ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:27: relocating range for 9 to store 4
    changefeed_dist_test.go:437: 2024-08-09 18:10:27: relocating range for 9 to store 4 took 370.086198ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:27: relocating range for 10 to store 4
    changefeed_dist_test.go:437: 2024-08-09 18:10:27: relocating range for 10 to store 4 took 366.490932ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:27: relocating range for 11 to store 4
    changefeed_dist_test.go:437: 2024-08-09 18:10:28: relocating range for 11 to store 4 took 347.625943ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:28: relocating range for 12 to store 4
    changefeed_dist_test.go:437: 2024-08-09 18:10:28: relocating range for 12 to store 4 took 448.654913ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:28: relocating range for 13 to store 4
    changefeed_dist_test.go:437: 2024-08-09 18:10:29: relocating range for 13 to store 4 took 265.231962ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:29: relocating range for 14 to store 4
    changefeed_dist_test.go:437: 2024-08-09 18:10:29: relocating range for 14 to store 4 took 351.528157ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:29: relocating range for 15 to store 4
    changefeed_dist_test.go:437: 2024-08-09 18:10:29: relocating range for 15 to store 4 took 322.302968ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:29: relocating range for 16 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:30: relocating range for 16 to store 5 took 368.735694ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:30: relocating range for 17 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:30: relocating range for 17 to store 5 took 378.519316ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:30: relocating range for 18 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:30: relocating range for 18 to store 5 took 379.509217ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:30: relocating range for 19 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:31: relocating range for 19 to store 5 took 384.748835ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:31: relocating range for 20 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:31: relocating range for 20 to store 5 took 259.727657ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:31: relocating range for 21 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:31: relocating range for 21 to store 5 took 401.03545ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:31: relocating range for 22 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:32: relocating range for 22 to store 5 took 342.270707ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:32: relocating range for 23 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:32: relocating range for 23 to store 5 took 305.868312ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:32: relocating range for 24 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:32: relocating range for 24 to store 5 took 365.047616ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:32: relocating range for 25 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:33: relocating range for 25 to store 5 took 305.197938ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:33: relocating range for 26 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:33: relocating range for 26 to store 5 took 337.708121ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:33: relocating range for 27 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:33: relocating range for 27 to store 5 took 324.839698ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:33: relocating range for 28 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:34: relocating range for 28 to store 5 took 279.655234ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:34: relocating range for 29 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:34: relocating range for 29 to store 5 took 396.654322ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:34: relocating range for 30 to store 5
    changefeed_dist_test.go:437: 2024-08-09 18:10:34: relocating range for 30 to store 5 took 278.234674ms
    changefeed_dist_test.go:432: 2024-08-09 18:10:34: relocating range for 31 to store 5
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16399218?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16399218?buildTab=artifacts#/) on master @ [19380bb6e4cf1e1f2a6335cbceb2ce962d4bbd65](https://github.com/cockroachdb/cockroach/commits/19380bb6e4cf1e1f2a6335cbceb2ce962d4bbd65):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 3928340 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 0 to store 1 took 5.58528ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 1 to store 1
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 1 to store 1 took 6.132726ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 2 to store 2
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 2 to store 2 took 41.78757ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 3 to store 2
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 3 to store 2 took 74.564787ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 4 to store 3
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 4 to store 3 took 132.133299ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 5 to store 3
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 5 to store 3 took 85.343546ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 6 to store 3
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 6 to store 3 took 41.057204ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 7 to store 3
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 7 to store 3 took 125.323892ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 8 to store 4
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 8 to store 4 took 50.668329ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 9 to store 4
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 9 to store 4 took 61.395459ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 10 to store 4
    changefeed_dist_test.go:437: 2024-08-09 19:05:36: relocating range for 10 to store 4 took 43.579372ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:36: relocating range for 11 to store 4
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 11 to store 4 took 159.817109ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 12 to store 4
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 12 to store 4 took 44.978502ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 13 to store 4
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 13 to store 4 took 48.546778ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 14 to store 4
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 14 to store 4 took 53.342818ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 15 to store 4
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 15 to store 4 took 91.586707ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 16 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 16 to store 5 took 103.949917ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 17 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 17 to store 5 took 118.681915ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 18 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 18 to store 5 took 141.193503ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 19 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 19 to store 5 took 69.325155ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 20 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:37: relocating range for 20 to store 5 took 127.474628ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:37: relocating range for 21 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:38: relocating range for 21 to store 5 took 81.735409ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:38: relocating range for 22 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:38: relocating range for 22 to store 5 took 168.166932ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:38: relocating range for 23 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:38: relocating range for 23 to store 5 took 92.032166ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:38: relocating range for 24 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:05:38: relocating range for 24 to store 5 took 94.608246ms
    changefeed_dist_test.go:432: 2024-08-09 19:05:38: relocating range for 25 to store 5
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16400150?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16400150?buildTab=artifacts#/) on master @ [19380bb6e4cf1e1f2a6335cbceb2ce962d4bbd65](https://github.com/cockroachdb/cockroach/commits/19380bb6e4cf1e1f2a6335cbceb2ce962d4bbd65):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 8325657 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 29 to store 5 took 33.228649ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 30 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 30 to store 5 took 31.572394ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 31 to store 5
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 31 to store 5 took 28.324048ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 32 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 32 to store 6 took 27.487422ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 33 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 33 to store 6 took 27.504194ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 34 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 34 to store 6 took 27.222581ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 35 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 35 to store 6 took 27.346493ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 36 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 36 to store 6 took 27.64844ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 37 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 37 to store 6 took 26.895748ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 38 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 38 to store 6 took 33.505389ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 39 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 39 to store 6 took 27.225528ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 40 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 40 to store 6 took 27.731197ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 41 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 41 to store 6 took 28.086588ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 42 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 42 to store 6 took 27.475779ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 43 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 43 to store 6 took 27.774493ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 44 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 44 to store 6 took 27.496047ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 45 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 45 to store 6 took 28.251576ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 46 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 46 to store 6 took 27.8721ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 47 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 47 to store 6 took 32.25418ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 48 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 48 to store 6 took 28.220262ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 49 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 49 to store 6 took 28.098515ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 50 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 50 to store 6 took 27.347528ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 51 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 51 to store 6 took 27.829584ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 52 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 52 to store 6 took 27.569708ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 53 to store 6
    changefeed_dist_test.go:437: 2024-08-09 19:44:28: relocating range for 53 to store 6 took 28.0603ms
    changefeed_dist_test.go:432: 2024-08-09 19:44:28: relocating range for 54 to store 6
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16401797?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16401797?buildTab=artifacts#/) on master @ [f429ec2434ea777ec3a5d76b0503907b43b6a088](https://github.com/cockroachdb/cockroach/commits/f429ec2434ea777ec3a5d76b0503907b43b6a088):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 4585321 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 0 to store 1 took 9.125147ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 1 to store 1
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 1 to store 1 took 17.503536ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 2 to store 2
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 2 to store 2 took 74.57196ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 3 to store 2
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 3 to store 2 took 52.570602ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 4 to store 3
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 4 to store 3 took 68.759112ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 5 to store 3
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 5 to store 3 took 57.377094ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 6 to store 3
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 6 to store 3 took 64.520353ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 7 to store 3
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 7 to store 3 took 115.659676ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 8 to store 4
    changefeed_dist_test.go:437: 2024-08-09 21:07:02: relocating range for 8 to store 4 took 151.12448ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:02: relocating range for 9 to store 4
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 9 to store 4 took 66.466041ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 10 to store 4
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 10 to store 4 took 34.54363ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 11 to store 4
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 11 to store 4 took 107.522906ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 12 to store 4
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 12 to store 4 took 58.010002ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 13 to store 4
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 13 to store 4 took 71.737598ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 14 to store 4
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 14 to store 4 took 77.495778ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 15 to store 4
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 15 to store 4 took 90.613311ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 16 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 16 to store 5 took 58.691209ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 17 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 17 to store 5 took 56.33473ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 18 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 18 to store 5 took 54.803475ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 19 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 19 to store 5 took 41.679747ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 20 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 20 to store 5 took 107.4726ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 21 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 21 to store 5 took 43.176711ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 22 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:03: relocating range for 22 to store 5 took 136.433742ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:03: relocating range for 23 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:04: relocating range for 23 to store 5 took 113.17838ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:04: relocating range for 24 to store 5
    changefeed_dist_test.go:437: 2024-08-09 21:07:04: relocating range for 24 to store 5 took 138.177887ms
    changefeed_dist_test.go:432: 2024-08-09 21:07:04: relocating range for 25 to store 5
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16402922?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16402922?buildTab=artifacts#/) on master @ [c64fb80786b82b0a848bf59e271d8b2ce44db674](https://github.com/cockroachdb/cockroach/commits/c64fb80786b82b0a848bf59e271d8b2ce44db674):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 3797725 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1445666381
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-09 21:52:58: starting the test cluster took 7.758488132s
    changefeed_dist_test.go:410: 2024-08-09 21:52:59: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-09 21:52:59: creating and inserting rows into table took 146.286135ms
    changefeed_dist_test.go:418: 2024-08-09 21:52:59: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-09 21:53:00: spitting the table took 920.412489ms
    changefeed_dist_test.go:432: 2024-08-09 21:53:00: relocating range for 0 to store 1
    changefeed_dist_test.go:437: 2024-08-09 21:53:00: relocating range for 0 to store 1 took 7.886883ms
    changefeed_dist_test.go:432: 2024-08-09 21:53:00: relocating range for 1 to store 1
    changefeed_dist_test.go:437: 2024-08-09 21:53:00: relocating range for 1 to store 1 took 41.816188ms
    changefeed_dist_test.go:432: 2024-08-09 21:53:00: relocating range for 2 to store 2
    changefeed_dist_test.go:437: 2024-08-09 21:53:00: relocating range for 2 to store 2 took 324.517011ms
    changefeed_dist_test.go:432: 2024-08-09 21:53:00: relocating range for 3 to store 2
    changefeed_dist_test.go:437: 2024-08-09 21:53:00: relocating range for 3 to store 2 took 266.943684ms
    changefeed_dist_test.go:432: 2024-08-09 21:53:00: relocating range for 4 to store 3
    changefeed_dist_test.go:437: 2024-08-09 21:53:01: relocating range for 4 to store 3 took 328.135143ms
    changefeed_dist_test.go:432: 2024-08-09 21:53:01: relocating range for 5 to store 3
    changefeed_dist_test.go:437: 2024-08-09 21:53:01: relocating range for 5 to store 3 took 299.336759ms
    changefeed_dist_test.go:432: 2024-08-09 21:53:01: relocating range for 6 to store 3
    changefeed_dist_test.go:437: 2024-08-09 21:53:01: relocating range for 6 to store 3 took 305.316532ms
    changefeed_dist_test.go:432: 2024-08-09 21:53:01: relocating range for 7 to store 3
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16405011?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16405011?buildTab=artifacts#/) on master @ [27468a22f82fb1e500bf7b35a62ca58d80abac39](https://github.com/cockroachdb/cockroach/commits/27468a22f82fb1e500bf7b35a62ca58d80abac39):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5526280 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:423: 2024-08-10 04:26:55: spitting the table took 658.723079ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:55: relocating range for 0 to store 1
    changefeed_dist_test.go:437: 2024-08-10 04:26:55: relocating range for 0 to store 1 took 5.729387ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:55: relocating range for 1 to store 1
    changefeed_dist_test.go:437: 2024-08-10 04:26:55: relocating range for 1 to store 1 took 4.640671ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:55: relocating range for 2 to store 2
    changefeed_dist_test.go:437: 2024-08-10 04:26:55: relocating range for 2 to store 2 took 43.705673ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:55: relocating range for 3 to store 2
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 3 to store 2 took 92.009157ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 4 to store 3
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 4 to store 3 took 109.325721ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 5 to store 3
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 5 to store 3 took 28.555909ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 6 to store 3
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 6 to store 3 took 40.965061ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 7 to store 3
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 7 to store 3 took 95.257145ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 8 to store 4
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 8 to store 4 took 121.064321ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 9 to store 4
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 9 to store 4 took 43.114744ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 10 to store 4
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 10 to store 4 took 41.489587ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 11 to store 4
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 11 to store 4 took 29.162417ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 12 to store 4
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 12 to store 4 took 28.340779ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 13 to store 4
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 13 to store 4 took 39.730739ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 14 to store 4
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 14 to store 4 took 40.4606ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 15 to store 4
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 15 to store 4 took 80.355934ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 16 to store 5
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 16 to store 5 took 51.163704ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 17 to store 5
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 17 to store 5 took 39.837069ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 18 to store 5
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 18 to store 5 took 28.675013ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 19 to store 5
    changefeed_dist_test.go:437: 2024-08-10 04:26:56: relocating range for 19 to store 5 took 48.320087ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:56: relocating range for 20 to store 5
    changefeed_dist_test.go:437: 2024-08-10 04:26:57: relocating range for 20 to store 5 took 84.326013ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:57: relocating range for 21 to store 5
    changefeed_dist_test.go:437: 2024-08-10 04:26:57: relocating range for 21 to store 5 took 31.788862ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:57: relocating range for 22 to store 5
    changefeed_dist_test.go:437: 2024-08-10 04:26:57: relocating range for 22 to store 5 took 89.261279ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:57: relocating range for 23 to store 5
    changefeed_dist_test.go:437: 2024-08-10 04:26:57: relocating range for 23 to store 5 took 48.879969ms
    changefeed_dist_test.go:432: 2024-08-10 04:26:57: relocating range for 24 to store 5
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16422961?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16422961?buildTab=artifacts#/) on master @ [2f0b0399a5c9656bdf8a417d0ab57e4eb1ae0f64](https://github.com/cockroachdb/cockroach/commits/2f0b0399a5c9656bdf8a417d0ab57e4eb1ae0f64):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 7740397 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3041535226
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-12 00:15:00: starting the test cluster took 4.683667405s
    changefeed_dist_test.go:410: 2024-08-12 00:15:01: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-12 00:15:01: creating and inserting rows into table took 40.579448ms
    changefeed_dist_test.go:418: 2024-08-12 00:15:01: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-12 00:15:01: spitting the table took 608.473159ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:01: relocating range for 0 to store 1
    changefeed_dist_test.go:437: 2024-08-12 00:15:01: relocating range for 0 to store 1 took 7.462217ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:01: relocating range for 1 to store 1
    changefeed_dist_test.go:437: 2024-08-12 00:15:01: relocating range for 1 to store 1 took 42.366086ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:01: relocating range for 2 to store 2
    changefeed_dist_test.go:437: 2024-08-12 00:15:01: relocating range for 2 to store 2 took 261.396548ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:01: relocating range for 3 to store 2
    changefeed_dist_test.go:437: 2024-08-12 00:15:02: relocating range for 3 to store 2 took 257.379834ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:02: relocating range for 4 to store 3
    changefeed_dist_test.go:437: 2024-08-12 00:15:02: relocating range for 4 to store 3 took 242.654709ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:02: relocating range for 5 to store 3
    changefeed_dist_test.go:437: 2024-08-12 00:15:02: relocating range for 5 to store 3 took 265.659004ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:02: relocating range for 6 to store 3
    changefeed_dist_test.go:437: 2024-08-12 00:15:03: relocating range for 6 to store 3 took 298.833245ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:03: relocating range for 7 to store 3
    changefeed_dist_test.go:437: 2024-08-12 00:15:03: relocating range for 7 to store 3 took 278.876136ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:03: relocating range for 8 to store 4
    changefeed_dist_test.go:437: 2024-08-12 00:15:03: relocating range for 8 to store 4 took 297.402846ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:03: relocating range for 9 to store 4
    changefeed_dist_test.go:437: 2024-08-12 00:15:03: relocating range for 9 to store 4 took 266.10914ms
    changefeed_dist_test.go:432: 2024-08-12 00:15:03: relocating range for 10 to store 4
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16440281?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16440281?buildTab=artifacts#/) on master @ [6bae06f4d458816874acf0faf1fc1cec2e099ac9](https://github.com/cockroachdb/cockroach/commits/6bae06f4d458816874acf0faf1fc1cec2e099ac9):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m51s)

```
Stack: 

```
goroutine 7154032 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    changefeed_dist_test.go:437: 2024-08-12 22:37:01: relocating range for 0 to store 1 took 27.678599ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:01: relocating range for 1 to store 1
    changefeed_dist_test.go:437: 2024-08-12 22:37:01: relocating range for 1 to store 1 took 13.735742ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:01: relocating range for 2 to store 2
    changefeed_dist_test.go:437: 2024-08-12 22:37:01: relocating range for 2 to store 2 took 52.580108ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:01: relocating range for 3 to store 2
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 3 to store 2 took 59.716382ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 4 to store 3
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 4 to store 3 took 66.794443ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 5 to store 3
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 5 to store 3 took 33.361762ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 6 to store 3
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 6 to store 3 took 114.186929ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 7 to store 3
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 7 to store 3 took 114.174347ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 8 to store 4
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 8 to store 4 took 102.432158ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 9 to store 4
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 9 to store 4 took 34.121005ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 10 to store 4
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 10 to store 4 took 163.68001ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 11 to store 4
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 11 to store 4 took 94.526054ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 12 to store 4
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 12 to store 4 took 131.819366ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 13 to store 4
    changefeed_dist_test.go:437: 2024-08-12 22:37:02: relocating range for 13 to store 4 took 59.388299ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:02: relocating range for 14 to store 4
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 14 to store 4 took 85.686381ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 15 to store 4
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 15 to store 4 took 55.193523ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 16 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 16 to store 5 took 95.223154ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 17 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 17 to store 5 took 40.097705ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 18 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 18 to store 5 took 133.533499ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 19 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 19 to store 5 took 74.360215ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 20 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 20 to store 5 took 80.883343ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 21 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 21 to store 5 took 40.046478ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 22 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 22 to store 5 took 129.784503ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 23 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 23 to store 5 took 35.938979ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 24 to store 5
    changefeed_dist_test.go:437: 2024-08-12 22:37:03: relocating range for 24 to store 5 took 172.590408ms
    changefeed_dist_test.go:432: 2024-08-12 22:37:03: relocating range for 25 to store 5
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16455449?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16455449?buildTab=artifacts#/) on master @ [66f6567d729eae799a87cb1ca89375fb1179a01b](https://github.com/cockroachdb/cockroach/commits/66f6567d729eae799a87cb1ca89375fb1179a01b):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m48s)

```
Stack: 

```
goroutine 5176047 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3673195789
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-13 21:13:45: starting the test cluster took 4.638951772s
    changefeed_dist_test.go:410: 2024-08-13 21:13:45: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-13 21:13:45: creating and inserting rows into table took 61.850459ms
    changefeed_dist_test.go:418: 2024-08-13 21:13:45: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-13 21:13:45: spitting the table took 740.021305ms
    changefeed_dist_test.go:426: 2024-08-13 21:13:45: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16472824?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16472824?buildTab=artifacts#/) on master @ [94211495de7788b3155bbe3070a9e7004c56538a](https://github.com/cockroachdb/cockroach/commits/94211495de7788b3155bbe3070a9e7004c56538a):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 8989610 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality322818188
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-14 20:56:12: starting the test cluster took 7.600714154s
    changefeed_dist_test.go:410: 2024-08-14 20:56:12: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-14 20:56:12: creating and inserting rows into table took 156.416399ms
    changefeed_dist_test.go:418: 2024-08-14 20:56:12: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-14 20:56:13: spitting the table took 1.027368793s
    changefeed_dist_test.go:426: 2024-08-14 20:56:13: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16477500?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16477500?buildTab=artifacts#/) on master @ [4a293c61143a7155fae1cf8a619beb8e43a7c434](https://github.com/cockroachdb/cockroach/commits/4a293c61143a7155fae1cf8a619beb8e43a7c434):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5894486 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality890107582
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-15 08:56:28: starting the test cluster took 4.900765769s
    changefeed_dist_test.go:410: 2024-08-15 08:56:29: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-15 08:56:29: creating and inserting rows into table took 71.182699ms
    changefeed_dist_test.go:418: 2024-08-15 08:56:29: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-15 08:56:29: spitting the table took 752.379373ms
    changefeed_dist_test.go:426: 2024-08-15 08:56:29: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16483532?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16483532?buildTab=artifacts#/) on master @ [2760ba0f283e753203ce77130c75ec0b69ac125d](https://github.com/cockroachdb/cockroach/commits/2760ba0f283e753203ce77130c75ec0b69ac125d):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 4008443 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3600240629
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-15 15:45:21: starting the test cluster took 5.0753498s
    changefeed_dist_test.go:410: 2024-08-15 15:45:21: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-15 15:45:22: creating and inserting rows into table took 65.700934ms
    changefeed_dist_test.go:418: 2024-08-15 15:45:22: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-15 15:45:22: spitting the table took 786.943935ms
    changefeed_dist_test.go:426: 2024-08-15 15:45:22: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot P-2 T-cdc branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16504096?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16504096?buildTab=artifacts#/) on master @ [0c74c3f9a7bc0cdc0a8f301d1a081f64da8d9da5](https://github.com/cockroachdb/cockroach/commits/0c74c3f9a7bc0cdc0a8f301d1a081f64da8d9da5):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5483755 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1868883344
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-16 20:21:13: starting the test cluster took 6.648219194s
    changefeed_dist_test.go:410: 2024-08-16 20:21:13: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-16 20:21:13: creating and inserting rows into table took 82.512385ms
    changefeed_dist_test.go:418: 2024-08-16 20:21:13: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-16 20:21:14: spitting the table took 820.153989ms
    changefeed_dist_test.go:426: 2024-08-16 20:21:14: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16534358?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16534358?buildTab=artifacts#/) on master @ [97dab43b1760f4ee316171c8db884b27121e3695](https://github.com/cockroachdb/cockroach/commits/97dab43b1760f4ee316171c8db884b27121e3695):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 4787015 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1532843240
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-19 16:03:18: starting the test cluster took 5.401013435s
    changefeed_dist_test.go:410: 2024-08-19 16:03:18: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-19 16:03:18: creating and inserting rows into table took 78.852706ms
    changefeed_dist_test.go:418: 2024-08-19 16:03:18: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-19 16:03:19: spitting the table took 849.980648ms
    changefeed_dist_test.go:426: 2024-08-19 16:03:19: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16549243?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16549243?buildTab=artifacts#/) on master @ [8296213abcc67caaf5f245ac747f240caa1f07d9](https://github.com/cockroachdb/cockroach/commits/8296213abcc67caaf5f245ac747f240caa1f07d9):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5553061 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality813413851
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-20 15:07:59: starting the test cluster took 6.778913228s
    changefeed_dist_test.go:410: 2024-08-20 15:07:59: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-20 15:07:59: creating and inserting rows into table took 220.392036ms
    changefeed_dist_test.go:418: 2024-08-20 15:08:00: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-20 15:08:00: spitting the table took 794.486119ms
    changefeed_dist_test.go:426: 2024-08-20 15:08:00: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16552664?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16552664?buildTab=artifacts#/) on master @ [85e85d72e6684fbef729c577b440cf2cecdf4326](https://github.com/cockroachdb/cockroach/commits/85e85d72e6684fbef729c577b440cf2cecdf4326):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 7683668 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2805717555
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-20 18:12:23: starting the test cluster took 7.514942315s
    changefeed_dist_test.go:410: 2024-08-20 18:12:23: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-20 18:12:23: creating and inserting rows into table took 278.607988ms
    changefeed_dist_test.go:418: 2024-08-20 18:12:23: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-20 18:12:25: spitting the table took 1.089388017s
    changefeed_dist_test.go:426: 2024-08-20 18:12:25: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16553067?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16553067?buildTab=artifacts#/) on master @ [333e528073c8f7a0d6477d75b1b4c7df67777cdb](https://github.com/cockroachdb/cockroach/commits/333e528073c8f7a0d6477d75b1b4c7df67777cdb):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5493790 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2238435386
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-20 18:10:47: starting the test cluster took 5.67896452s
    changefeed_dist_test.go:410: 2024-08-20 18:10:47: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-20 18:10:47: creating and inserting rows into table took 65.668363ms
    changefeed_dist_test.go:418: 2024-08-20 18:10:47: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-20 18:10:48: spitting the table took 921.909786ms
    changefeed_dist_test.go:426: 2024-08-20 18:10:48: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16556236?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16556236?buildTab=artifacts#/) on master @ [ebefe416f4389f496db56c1c56898b084d1e23d1](https://github.com/cockroachdb/cockroach/commits/ebefe416f4389f496db56c1c56898b084d1e23d1):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 3679009 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality130043631
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-20 20:49:47: starting the test cluster took 4.392499909s
    changefeed_dist_test.go:410: 2024-08-20 20:49:47: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-20 20:49:47: creating and inserting rows into table took 30.782298ms
    changefeed_dist_test.go:418: 2024-08-20 20:49:47: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-20 20:49:47: spitting the table took 523.844565ms
    changefeed_dist_test.go:426: 2024-08-20 20:49:47: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16563107?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16563107?buildTab=artifacts#/) on master @ [dafb8b704496a3caa241ee0a1dd12e9011699476](https://github.com/cockroachdb/cockroach/commits/dafb8b704496a3caa241ee0a1dd12e9011699476):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 6328445 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1274593606
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-21 13:13:47: starting the test cluster took 5.796010968s
    changefeed_dist_test.go:410: 2024-08-21 13:13:47: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-21 13:13:47: creating and inserting rows into table took 49.500915ms
    changefeed_dist_test.go:418: 2024-08-21 13:13:47: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-21 13:13:48: spitting the table took 868.609114ms
    changefeed_dist_test.go:426: 2024-08-21 13:13:48: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16593272?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16593272?buildTab=artifacts#/) on master @ [b1359ae64de710bcd928f5e0fef91a59a8f4a309](https://github.com/cockroachdb/cockroach/commits/b1359ae64de710bcd928f5e0fef91a59a8f4a309):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 6169507 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3037235330
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-23 14:44:11: starting the test cluster took 6.607408151s
    changefeed_dist_test.go:410: 2024-08-23 14:44:11: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-23 14:44:11: creating and inserting rows into table took 74.602516ms
    changefeed_dist_test.go:418: 2024-08-23 14:44:11: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-23 14:44:12: spitting the table took 1.07120386s
    changefeed_dist_test.go:426: 2024-08-23 14:44:12: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16596478?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16596478?buildTab=artifacts#/) on master @ [aaca0ec799b328e6ba7a738c28beb7fcece06217](https://github.com/cockroachdb/cockroach/commits/aaca0ec799b328e6ba7a738c28beb7fcece06217):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5187119 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3623561245
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-23 18:40:49: starting the test cluster took 5.822185894s
    changefeed_dist_test.go:410: 2024-08-23 18:40:49: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-23 18:40:49: creating and inserting rows into table took 66.271877ms
    changefeed_dist_test.go:418: 2024-08-23 18:40:49: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-23 18:40:50: spitting the table took 787.222739ms
    changefeed_dist_test.go:426: 2024-08-23 18:40:50: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16597637?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16597637?buildTab=artifacts#/) on master @ [5cbe091de105b5d8f7e6eccec5c72987702679ab](https://github.com/cockroachdb/cockroach/commits/5cbe091de105b5d8f7e6eccec5c72987702679ab):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 6921226 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1659663146
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-23 20:04:01: starting the test cluster took 6.914141162s
    changefeed_dist_test.go:410: 2024-08-23 20:04:02: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-23 20:04:02: creating and inserting rows into table took 109.382571ms
    changefeed_dist_test.go:418: 2024-08-23 20:04:02: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-23 20:04:03: spitting the table took 1.074486468s
    changefeed_dist_test.go:426: 2024-08-23 20:04:03: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16598409?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16598409?buildTab=artifacts#/) on master @ [5cbe091de105b5d8f7e6eccec5c72987702679ab](https://github.com/cockroachdb/cockroach/commits/5cbe091de105b5d8f7e6eccec5c72987702679ab):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8339773 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3723932009
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-23 20:34:13: starting the test cluster took 3.56240415s
    changefeed_dist_test.go:410: 2024-08-23 20:34:14: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-23 20:34:14: creating and inserting rows into table took 25.641595ms
    changefeed_dist_test.go:418: 2024-08-23 20:34:14: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-23 20:34:14: spitting the table took 460.51771ms
    changefeed_dist_test.go:426: 2024-08-23 20:34:14: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16598831?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16598831?buildTab=artifacts#/) on master @ [5cbe091de105b5d8f7e6eccec5c72987702679ab](https://github.com/cockroachdb/cockroach/commits/5cbe091de105b5d8f7e6eccec5c72987702679ab):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 8300981 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1938023142
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-23 20:56:47: starting the test cluster took 3.603477287s
    changefeed_dist_test.go:410: 2024-08-23 20:56:47: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-23 20:56:47: creating and inserting rows into table took 35.620442ms
    changefeed_dist_test.go:418: 2024-08-23 20:56:47: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-23 20:56:47: spitting the table took 461.855813ms
    changefeed_dist_test.go:426: 2024-08-23 20:56:47: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16627890?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16627890?buildTab=artifacts#/) on master @ [6eb869f92c1bed72563848c9f41ca4640e7f300f](https://github.com/cockroachdb/cockroach/commits/6eb869f92c1bed72563848c9f41ca4640e7f300f):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5706295 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2209457558
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-27 01:30:13: starting the test cluster took 5.399998665s
    changefeed_dist_test.go:410: 2024-08-27 01:30:13: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-27 01:30:13: creating and inserting rows into table took 80.19872ms
    changefeed_dist_test.go:418: 2024-08-27 01:30:13: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-27 01:30:14: spitting the table took 911.985398ms
    changefeed_dist_test.go:426: 2024-08-27 01:30:14: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16632391?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16632391?buildTab=artifacts#/) on master @ [9027e081ccc98fd344402d6a5db5eb53ebfc9efe](https://github.com/cockroachdb/cockroach/commits/9027e081ccc98fd344402d6a5db5eb53ebfc9efe):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5666245 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3992945266
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-27 10:54:45: starting the test cluster took 4.603590132s
    changefeed_dist_test.go:410: 2024-08-27 10:54:46: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-27 10:54:46: creating and inserting rows into table took 63.669955ms
    changefeed_dist_test.go:418: 2024-08-27 10:54:46: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-27 10:54:46: spitting the table took 805.704737ms
    changefeed_dist_test.go:426: 2024-08-27 10:54:46: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16637721?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16637721?buildTab=artifacts#/) on master @ [eada67c69b606999055e69a5e313581cad6b84c3](https://github.com/cockroachdb/cockroach/commits/eada67c69b606999055e69a5e313581cad6b84c3):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5413712 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality960963078
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-27 17:28:58: starting the test cluster took 5.636435568s
    changefeed_dist_test.go:410: 2024-08-27 17:28:58: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-27 17:28:59: creating and inserting rows into table took 122.393901ms
    changefeed_dist_test.go:418: 2024-08-27 17:28:59: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-27 17:28:59: spitting the table took 785.621375ms
    changefeed_dist_test.go:426: 2024-08-27 17:28:59: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16642180?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16642180?buildTab=artifacts#/) on master @ [4d78390e719359bfabb3046d2b1807bb0f09d84a](https://github.com/cockroachdb/cockroach/commits/4d78390e719359bfabb3046d2b1807bb0f09d84a):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 6115171 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3326705723
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-27 20:42:27: starting the test cluster took 5.561360236s
    changefeed_dist_test.go:410: 2024-08-27 20:42:27: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-27 20:42:28: creating and inserting rows into table took 129.939198ms
    changefeed_dist_test.go:418: 2024-08-27 20:42:28: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-27 20:42:28: spitting the table took 849.781085ms
    changefeed_dist_test.go:426: 2024-08-27 20:42:28: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16651041?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16651041?buildTab=artifacts#/) on master @ [b2f2e2042942c40f0a4c86be9a8f61ff0a6db6fc](https://github.com/cockroachdb/cockroach/commits/b2f2e2042942c40f0a4c86be9a8f61ff0a6db6fc):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 5502672 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2615577509
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-28 12:23:02: starting the test cluster took 4.806049398s
    changefeed_dist_test.go:410: 2024-08-28 12:23:02: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-28 12:23:02: creating and inserting rows into table took 54.854388ms
    changefeed_dist_test.go:418: 2024-08-28 12:23:02: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-28 12:23:03: spitting the table took 743.061476ms
    changefeed_dist_test.go:426: 2024-08-28 12:23:03: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16651855?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16651855?buildTab=artifacts#/) on master @ [b2f2e2042942c40f0a4c86be9a8f61ff0a6db6fc](https://github.com/cockroachdb/cockroach/commits/b2f2e2042942c40f0a4c86be9a8f61ff0a6db6fc):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m55s)

```
Stack: 

```
goroutine 8349274 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3471742079
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-28 13:06:29: starting the test cluster took 3.550707551s
    changefeed_dist_test.go:410: 2024-08-28 13:06:29: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-28 13:06:29: creating and inserting rows into table took 27.097679ms
    changefeed_dist_test.go:418: 2024-08-28 13:06:29: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-28 13:06:29: spitting the table took 467.924524ms
    changefeed_dist_test.go:426: 2024-08-28 13:06:29: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16663842?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16663842?buildTab=artifacts#/) on master @ [74429f0c2f2e54bd1aecd5d59776887d1755be8a](https://github.com/cockroachdb/cockroach/commits/74429f0c2f2e54bd1aecd5d59776887d1755be8a):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 4413117 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3325444329
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-29 06:05:43: starting the test cluster took 4.884709849s
    changefeed_dist_test.go:410: 2024-08-29 06:05:43: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-29 06:05:44: creating and inserting rows into table took 45.548441ms
    changefeed_dist_test.go:418: 2024-08-29 06:05:44: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-29 06:05:44: spitting the table took 780.090273ms
    changefeed_dist_test.go:426: 2024-08-29 06:05:44: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16670198?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16670198?buildTab=artifacts#/) on master @ [67b61e597f9feb50a9253ae38e7fc9b53406938c](https://github.com/cockroachdb/cockroach/commits/67b61e597f9feb50a9253ae38e7fc9b53406938c):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 6169028 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3992817546
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-29 15:33:36: starting the test cluster took 6.865700499s
    changefeed_dist_test.go:410: 2024-08-29 15:33:36: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-29 15:33:37: creating and inserting rows into table took 176.349476ms
    changefeed_dist_test.go:418: 2024-08-29 15:33:37: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-29 15:33:38: spitting the table took 956.688083ms
    changefeed_dist_test.go:426: 2024-08-29 15:33:38: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16673262?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16673262?buildTab=artifacts#/) on master @ [65fdb8a90e7f2215bec5d3661056fcb3fa87084a](https://github.com/cockroachdb/cockroach/commits/65fdb8a90e7f2215bec5d3661056fcb3fa87084a):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 2869008 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2944314323
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-29 17:30:37: starting the test cluster took 8.025893001s
    changefeed_dist_test.go:410: 2024-08-29 17:30:37: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-29 17:30:37: creating and inserting rows into table took 137.335434ms
    changefeed_dist_test.go:418: 2024-08-29 17:30:37: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-29 17:30:38: spitting the table took 1.245607538s
    changefeed_dist_test.go:426: 2024-08-29 17:30:38: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16679343?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16679343?buildTab=artifacts#/) on master @ [4756b021edfdd477427f4ec34bf83f837f26a855](https://github.com/cockroachdb/cockroach/commits/4756b021edfdd477427f4ec34bf83f837f26a855):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 5310690 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality887410161
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-29 22:41:12: starting the test cluster took 8.165603974s
    changefeed_dist_test.go:410: 2024-08-29 22:41:12: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-29 22:41:12: creating and inserting rows into table took 181.886265ms
    changefeed_dist_test.go:418: 2024-08-29 22:41:12: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-29 22:41:13: spitting the table took 1.156396115s
    changefeed_dist_test.go:426: 2024-08-29 22:41:13: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16679861?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16679861?buildTab=artifacts#/) on master @ [4756b021edfdd477427f4ec34bf83f837f26a855](https://github.com/cockroachdb/cockroach/commits/4756b021edfdd477427f4ec34bf83f837f26a855):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8220049 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1922960011
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-29 23:11:40: starting the test cluster took 3.595440923s
    changefeed_dist_test.go:410: 2024-08-29 23:11:40: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-29 23:11:40: creating and inserting rows into table took 28.88903ms
    changefeed_dist_test.go:418: 2024-08-29 23:11:40: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-29 23:11:41: spitting the table took 480.238165ms
    changefeed_dist_test.go:426: 2024-08-29 23:11:41: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16679968?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16679968?buildTab=artifacts#/) on master @ [4756b021edfdd477427f4ec34bf83f837f26a855](https://github.com/cockroachdb/cockroach/commits/4756b021edfdd477427f4ec34bf83f837f26a855):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 8172482 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3843837540
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-29 23:34:27: starting the test cluster took 3.606354187s
    changefeed_dist_test.go:410: 2024-08-29 23:34:27: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-29 23:34:27: creating and inserting rows into table took 32.853678ms
    changefeed_dist_test.go:418: 2024-08-29 23:34:27: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-29 23:34:28: spitting the table took 474.280797ms
    changefeed_dist_test.go:426: 2024-08-29 23:34:28: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16690386?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16690386?buildTab=artifacts#/) on master @ [e1ec4041a14be9d885e5d1cd3b504ac814c6bbc7](https://github.com/cockroachdb/cockroach/commits/e1ec4041a14be9d885e5d1cd3b504ac814c6bbc7):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 3831751 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3542786732
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-30 18:16:36: starting the test cluster took 6.255297771s
    changefeed_dist_test.go:410: 2024-08-30 18:16:36: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-30 18:16:36: creating and inserting rows into table took 64.721151ms
    changefeed_dist_test.go:418: 2024-08-30 18:16:36: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-30 18:16:37: spitting the table took 1.072074914s
    changefeed_dist_test.go:426: 2024-08-30 18:16:37: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16695136?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16695136?buildTab=artifacts#/) on master @ [8551145a0c99c4c95a28ec470e699d0c20ca97ab](https://github.com/cockroachdb/cockroach/commits/8551145a0c99c4c95a28ec470e699d0c20ca97ab):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m52s)

```
Stack: 

```
goroutine 5053829 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality3203829493
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-08-31 04:30:09: starting the test cluster took 5.535765609s
    changefeed_dist_test.go:410: 2024-08-31 04:30:09: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-08-31 04:30:09: creating and inserting rows into table took 83.56765ms
    changefeed_dist_test.go:418: 2024-08-31 04:30:09: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-08-31 04:30:10: spitting the table took 780.555227ms
    changefeed_dist_test.go:426: 2024-08-31 04:30:10: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16727026?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16727026?buildTab=artifacts#/) on master @ [6318706297b37bcec52dd7524661d66b2487a4ab](https://github.com/cockroachdb/cockroach/commits/6318706297b37bcec52dd7524661d66b2487a4ab):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 5123060 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2197303465
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-09-03 06:27:37: starting the test cluster took 5.999863045s
    changefeed_dist_test.go:410: 2024-09-03 06:27:38: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-09-03 06:27:38: creating and inserting rows into table took 44.893717ms
    changefeed_dist_test.go:418: 2024-09-03 06:27:38: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-09-03 06:27:38: spitting the table took 844.027192ms
    changefeed_dist_test.go:426: 2024-09-03 06:27:38: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #129994 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc C-test-failure O-robot T-kv X-duplicate branch-release-24.2.1-rc]
- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-cdc A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16744238?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16744238?buildTab=artifacts#/) on master @ [ae386bd58eb341f2c3e8f95fc44ecbbac62cf6c4](https://github.com/cockroachdb/cockroach/commits/ae386bd58eb341f2c3e8f95fc44ecbbac62cf6c4):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m53s)

```
Stack: 

```
goroutine 4518811 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality1240962743
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-09-04 06:29:30: starting the test cluster took 6.499019849s
    changefeed_dist_test.go:410: 2024-09-04 06:29:30: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-09-04 06:29:31: creating and inserting rows into table took 91.054363ms
    changefeed_dist_test.go:418: 2024-09-04 06:29:31: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-09-04 06:29:31: spitting the table took 854.143776ms
    changefeed_dist_test.go:426: 2024-09-04 06:29:31: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #129994 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [C-test-failure O-robot T-kv X-duplicate branch-release-24.2.1-rc]
- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

ccl/changefeedccl.TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16758021?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/16758021?buildTab=artifacts#/) on master @ [268e3b368f658f70633540b8e122f7538a805023](https://github.com/cockroachdb/cockroach/commits/268e3b368f658f70633540b8e122f7538a805023):

Fatal error:

```
panic: test timed out after 14m57s
running tests:
	TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality (14m54s)

```
Stack: 

```
goroutine 5093988 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/f236cfce7e32f4b492de4b4f1cd80c96/logTestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality2673000285
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:157: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    changefeed_dist_test.go:398: 2024-09-04 22:33:13: starting the test cluster took 6.314545897s
    changefeed_dist_test.go:410: 2024-09-04 22:33:13: creating and inserting rows into table
    changefeed_dist_test.go:416: 2024-09-04 22:33:13: creating and inserting rows into table took 49.980881ms
    changefeed_dist_test.go:418: 2024-09-04 22:33:13: splitting table into single-key ranges
    changefeed_dist_test.go:423: 2024-09-04 22:33:14: spitting the table took 880.437662ms
    changefeed_dist_test.go:426: 2024-09-04 22:33:14: relocating ranges in exponential distribution
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #129994 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [C-test-failure O-robot T-kv X-duplicate branch-release-24.2.1-rc]
- #127027 ccl/changefeedccl: TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality failed [A-multitenancy C-test-failure O-robot P-2 T-kv X-duplicate branch-release-24.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedWithSimpleDistributionStrategyAndConstrainedLocality.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/ccl/changefeedccl/changefeed_dist_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_dist_test.go
@@ -405,6 +405,12 @@ func newRangeDistributionTester(
 	systemDB.Exec(t, "SET CLUSTER SETTING kv.rangefeed.enabled = true")
 	if tc.StartedDefaultTestTenant() {
 		systemDB.Exec(t, `ALTER TENANT [$1] GRANT CAPABILITY can_admin_relocate_range=true`, serverutils.TestTenantID().ToUint64())
+		// Give 1,000,000 upfront tokens to the tenant, and keep the tokens per
+		// second rate to the default value of 10,000. This helps avoid throttling
+		// in the tests.
+		systemDB.Exec(t,
+			"SELECT crdb_internal.update_tenant_resource_limits($1::INT, 1000000, 10000, 0, now(), 0)",
+			serverutils.TestTenantID().ToUint64())
 	}
 
 	t.Logf("%s: creating and inserting rows into table", timeutil.Now().Format(time.DateTime))

